_posts/2016-06-22-hello-beercraft.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Maku brewing and Solita's passion pale ale
+author: Rinorragi
+excerpt: Solita ordered passion pale ale from Maku brewing and got introduction to their craftmenship
+tags: 
+- Beercraft
+---
+Solita wanted to have their own beer. We had our internal voting for the taste and name for the beer and "Intohimo" passion pale ale won the contest. At 21.6.2016 we got chance to taste 20l of the beer and make adjustments to it. All started with bus trip from Solita's office at Helsinki to Tuusula. We had great time already in the bus in discussion of projects and expectations of the upcoming beer. 
+
+Here is our passion pale ale in the glass. 
+![Tools](/img/hello-beercraft/glass_of_beer.jpg)
+
+## Maku brewing
+
+[Maku brewing](http://www.makubrewing.com/) is few years old finnish craft beer brewery. They are making great beers. Their IPA is most likely the best finnish IPA and it got some awards as well. Their amber ale is also great and it has been awarder as well. After fitting 20 nerds into a bus we got introductionary into their brewery by Henri Parviainen who is just leaving the IT business and starting to be full-day at brewery. 
```

Comment:
Jotenkin näin?: is couple years old finnish craft beer brewery making great beers. Few vs. couple.
awarder -> awarded
leaving the IT business for becoming full-day brewer (?)